### DISCUSSION TITLE
Welcome to hangzhou_house_knowledge Discussions!

### DISCUSSION THREAD

**Discussion by houshanren** [houshanren/hangzhou_house_knowledge#160] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shrekboss** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Anankeg** [comment]
邮件已收到，会尽快回复
